includes/class-debug.php

```diff
@@ -144,17 +129,11 @@ public static function log( $message, $type = 'string' ) {
 			) . PHP_EOL;
 
 			$file_path = self::get_file_path();
-
-			$content = '';
-			if ( $wp_filesystem->exists( $file_path ) ) {
-				if ( $wp_filesystem->is_readable( $file_path ) ) {
-					$content = $wp_filesystem->get_contents( $file_path );
-				}
-			}
 			$wp_filesystem->put_contents(
 				$file_path,
-				$formatted_message . $content,
-				FS_CHMOD_FILE
+				$formatted_message,
+				FS_CHMOD_FILE,
+				'a'
```

Comment:
Previously, the new message was prepended to the log file (`$formatted_message . $content`). With this change, the new message will be appended to the log file (`$formatted_message` + `'a'`).

This makes sense. I'm just noting this change in the log's output. This will cause one test to fail, which I'm happy to update and push to this PR before we merge it when everything else is ready.